apiserver/facades/client/client/status_test.go

```diff
@@ -377,6 +377,54 @@ func (s *statusUnitTestSuite) TestMigrationInProgress(c *gc.C) {
 	setAndCheckMigStatus("oh noes")
 }
 
+func (s *statusUnitTestSuite) TestRelationFiltered(c *gc.C) {
+	s1 := s.Factory.MakeApplication(c, &factory.ApplicationParams{
```

Comment:
not your fault, but I find it quite hard to read and understand this test, maybe I'll try again tomorrow.

Reply:
Added comments and renamed some variables. Don't know if it made it clearer for you but it reads like a novel to me now :)